### PR TITLE
[release/v1.0] release: v1.0.2

### PR DESCRIPTION
**Pull request by Xunzhuo**
after: https://github.com/envoyproxy/gateway/pull/3594

release: v1.0.2

+ pin envoyproxy to `envoyproxy/envoy:distroless-v1.29.5`
+ add release notes and update VERSION (for release workflow)

### CODE REVIEW

File: site/content/en/latest/releases/v1.0.2.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- Added proxy protocol always as first listenerFilter
+- Set ignoreCase for header matchers in extAuth
+- Added backend TLS SAN validation
+- Fixed ReplaceFullPath not working for root path (/)
+
+## Providers
+- Fixed duplicated xroutes are added to gatewayapi Resources
+- Fixed security policy reference grant from field type
+- Fixed Route extension filters with different types but the same name and namespace aren't correctly cached
+- Fixed secrets/configmap updates do not trigger a controller reconcile
```
Comment (arkodg) [inline site/content/en/latest/releases/v1.0.2.md:26]:
```suggestion
- Fixed secrets/configmap updates to trigger a controller reconcile by removing the generationChanged predicate
```

File: release-notes/v1.0.2.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      Added proxy protocol always as first listenerFilter
+      Set ignoreCase for header matchers in extAuth
+      Added backend TLS SAN validation
+      Fixed ReplaceFullPath not working for root path (/)
+  - area: providers
+    change: |
+      Fixed duplicated xroutes are added to gatewayapi Resources
+      Fixed security policy reference grant from field type
+      Fixed Route extension filters with different types but the same name and namespace aren't correctly cached
+      Fixed secrets/configmap updates do not trigger a controller reconcile
```
Comment (arkodg) [inline release-notes/v1.0.2.yaml:23]: same as https://github.com/envoyproxy/gateway/pull/3595/files#r1637184431

File: site/content/en/latest/releases/v1.0.2.md (hunk 1 above)
Comment (Xunzhuo) [reply site/content/en/latest/releases/v1.0.2.md]: Done

File: release-notes/v1.0.2.yaml (hunk 2 above)
Comment (Xunzhuo) [reply release-notes/v1.0.2.yaml]: Done

**Comment by arkodg** [comment]
Hey @Xunzhuo can we also update the release notes to include - " Updated EnvoyProxy to 1.29.5 ?

**Review by arkodg — APPROVED** [review]
LGTM thanks !

**Review by shawnh2 — APPROVED** [review]
LGTM!

**Pull request merged**